src/servers/src/http/otlp.rs

```diff
@@ -34,13 +43,18 @@ use super::header::{write_cost_header_map, CONTENT_TYPE_PROTOBUF};
 use crate::error::{self, Result};
 use crate::query_handler::OpenTelemetryProtocolHandlerRef;
 
+const OTLP_GREPTIME_PIPELINE_NAME_HEADER_NAME: &str = "x-greptime-pipeline-name";
+const OTLP_GREPTIME_PIPELINE_VERSION_HEADER_NAME: &str = "x-greptime-pipeline-version";
+const OTLP_GREPTIME_TABLE_NAME_HEADER_NAME: &str = "x-greptime-table-name";
```

Comment:
I suggest to rename all these three headers to add log, say

- `x-greptime-log-pipeline-name`
- `x-greptime-log-pipeline-version`
- `x-greptime-log-table-name`

Because in otel collectors and api/sdk, the headers may be configured for all of metrics, logs and traces. We have to be able to know the table name is for logs instead of traces.